lib/shippo/api/transformers/list.rb

```diff
@@ -51,7 +51,11 @@ def transform
 
         def detect_type_class(model_name)
           type = model_name.to_s.singularize.camelize
-          "Shippo::#{type}".constantize rescue nil
+          begin
+            "Shippo::#{type}".constantize
+          rescue Exception
```

Comment:
@piecehealth would it be good enough to rescue both StandardError and LoadError here? That seems to be only what you need and would not risk adding confusion or concern to inadvertently catching other types of exceptions. (SyntaxError, SignalException, etc.)